src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -287,9 +287,9 @@ class CompanyStep extends React.Component {
                     <View style={styles.mt4}>
                         <StatePicker
                             label={this.props.translate('companyStep.incorporationState')}
-                            onChange={value => this.clearErrorAndSetValue('incorporationState', value)}
+                            onInputChange={value => this.clearErrorAndSetValue('incorporationState', value)}
                             value={this.state.incorporationState}
-                            hasError={this.getErrors().incorporationState}
+                            errorText={this.getErrorText('incorporationState')}
```

Comment:
The `incorporationState` key is not defined [here](https://github.com/Expensify/App/blob/cee55cbf688e7ed7f9de61cbe3695852f83f33e6/src/pages/ReimbursementAccount/CompanyStep.js#L79-L88).